### PR TITLE
[cypress] Combine multiple tests under visualizations into single test to save running time

### PR DESCRIPTION
**Pull request by kristw**
Extended the work from #6007 

**tl;dr** Based on the hypothesis that is there is an overhead to start up browser for each test, this PR nests all vis tests another level of `describe()` in `all.js` and tell cypress to ignore the individual test files. 

- Make each visualization test file export a function instead of declaring a test immediately.
- Rename vis test files to `_xxx.js` and add cypress rule to ignore any file that begins with underscore (`_`). 
- Create `all.js` which wraps all the test under a single run using nested `describe()`

Good speed up both locally and on ci. First attempt on ci is 10mins compared to previous run of 15mins.

@graceguo-supercat @michellethomas @williaster

### CODE REVIEW

**Comment by kristw** [comment]
I am going to re-run to get more data point.

**Review by graceguo-supercat — APPROVED** [review]
LGTM
